assets/components/paymentButtons/payPalExpressButton/payPalExpressButton.jsx

```diff
@@ -59,6 +61,17 @@ function Button(props: PropTypes) {
 
   const PayPalButton = window.paypal.Button.driver('react', { React, ReactDOM });
 
+  if (props.disable) {
+    return (
+      <button
```

Comment:
Could this be inside the `component-paypal-button-checkout` div from below, with a className of `component-paypal-button-checkout__fake-pop-up-button`?